api/events.go

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 const (
+	ModuleACL         = "acl"
```

Comment:
The alternative (which I now think is better) is to temporarily stop caching download contracts until we simplify the worker cache. It feels a bit silly to introduce a new module when we're likely removing it again soon anyway. What do you think @ChrisSchinnerl ?